.github/workflows/build_runner_image.yml

```diff
@@ -21,12 +21,13 @@ name: Build and Version Runner Docker Image
 on:
   push:
     branches: ['master']
+    paths: ['.github/gh-actions-self-hosted-runners/arc/images/**']
```

Comment:
Should we also include this file in the paths?

Reply:
(to be clear, I mean `.github/workflows/build_runner_image.yml`)

Reply:
Good question. Do we want it to trigger when changing the workflow itself?

Should this be common for all workflows?

Reply:
> Its hard to confirm but from docs on pull_request_target trigger
This event runs in the context of the base of the pull request
which i think means that the workflow in target will be run instead of the incoming code. Think that makes it moot to run

This is for the push event to master, not for the pull request event so I think it should still run on the changed code.

Reply:
Basically, if we make a change to the workflow that impacts the container, I want that new container to get pushed after the PR is merged